localization/english/wc_character_interactions_l_english.yml

```diff
@@ -9,4 +9,6 @@
  nothing:0 "Nothing"
 
  TARGET_SCOURGE_JAILOR_REASON:1 "[target.GetShortUIName|U] is a [scourge_jailor|E]: $VALUE|=+0$"
- RECIPIENT_SCOURGE_JAILOR_REASON:1 "[recipient.GetShortUIName|U] is a [scourge_jailor|E]: $VALUE|=+0$"
\ No newline at end of file
+ RECIPIENT_SCOURGE_JAILOR_REASON:1 "[recipient.GetShortUIName|U] is a [scourge_jailor|E]: $VALUE|=+0$"
+ 
+ LICH_KING_VOICE:0 "Hears your voice in the head: $VALUE|=+0$"
```

Comment:
probably the lich king trait should be linked here for greater clarity

Reply:
> 

How?

Reply:
Hears [lich king(trait)] voice in the head ?

Reply:
Done in d801618c1fc4203afa18b879429c4f407421f56f.